### PR TITLE
Add nullbits Bit-C PRO board

### PR DESCRIPTION
**Pull request by jaygreco**
Add the upcoming nullbits Bit-C PRO RP2040 board. Tested and working on our engineering samples.

Relies on this pico-sdk PR: raspberrypi/pico-sdk#1149

### CODE REVIEW

File: ports/raspberrypi/boards/nullbits_bit_c_pro/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,9 @@
+USB_VID = 0x2E8A
+USB_PID = 0x6E61
```
Comment (dhalbert) [inline ports/raspberrypi/boards/nullbits_bit_c_pro/mpconfigboard.mk:2]: Hi, did you get the USB_PID from Raspberry Pi? I don't see it in the list here: https://github.com/raspberrypi/usb-pid/blob/main/Readme.md

**Review by dhalbert — COMMENTED** [review]
One q re USB PID

**Comment by jaygreco** [comment]
@dhalbert good catch! Let me check in with raspberry pi re: the PID. I'll give you a ping when I get it resolved. Thanks!

File: ports/raspberrypi/boards/nullbits_bit_c_pro/mpconfigboard.mk (hunk 1 above)
Comment (jaygreco) [reply ports/raspberrypi/boards/nullbits_bit_c_pro/mpconfigboard.mk]: @dhalbert Gordon from Raspberry Pi confirmed USB_PID = 0x1048 and that the table will be updated soon. I've updated the PR with the correct PID.

**Review by dhalbert — APPROVED** [review]
Thanks for confirming the PID!

**Pull request merged**